ca-certificates.spec

```diff
@@ -35,9 +35,11 @@ Name: ca-certificates
 # to have increasing version numbers. However, the new scheme will work, 
 # because all future versions will start with 2013 or larger.)
 
-Version: 2021.2.50
-Release: 3
-License: Public Domain
+Version: 2023.2.62_v7.0.401
+# for Rawhide, please always use release >= 2
+# for Fedora release branches, please use release < 2 (1.0, 1.1, ...)
```

Comment:
These have been omitted earlier and since we're not fedora could keep on doing so.

Reply:
Removed